automatic/winscp.install/winscp.install.nuspec

```diff
@@ -2,15 +2,15 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
```

Comment:
this you really intend to change the nuspec file for this package?
Please revert.